web/html/src/branding/css/susemanager/components/cveaudit.less

```diff
@@ -0,0 +1,10 @@
+.scan-data-indicator {
```

Comment:
This file doesn't belong here, the right place to put these styles is in a style module that's adjacent to the view where you use it. In this specific case for example `web/html/src/manager/audit/cveaudit/cveaudit.module.less`. This also requires changes to how you use the classname in code. You can see `web/html/src/components/table/SearchPanel.module.less` and `web/html/src/components/table/SearchPanel.tsx` for an example on how to use style modules.

The components folder is for very generic styles for things such as inputs, buttons, etc, not for view-specific styles.

Reply:
I messed something up while resolving conflicts but it should be removed.

Reply:
Okay, thanks for clarifying. In that case it might be good to double check if there's anything else that might've gone missing in this process. One option to avoid issues like this for long-running brances would be to squash the change sets once a certain part gets approved. This would make rebasing easier, but on the other hand reduce the amount of history that's available, so it's mostly a question of whether the trade off is worth it for you.